packages/salesforcedx-vscode-core/package.json

```diff
@@ -218,7 +218,7 @@
         },
         {
           "command": "sfdx.force.source.delete.current.file",
-          "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking && resourceLangId != 'forcesourcemanifest'"
+          "when": "sfdx:project_opened && resourceLangId != 'forcesourcemanifest' && editorIsOpen"
```

Comment:
Why add " && editorIsOpen" here?  Did removing only "sfdx:default_username_has_no_change_tracking && " not work correctly?

Reply:
This command is provided for the file opened in the editor. That's why added `&& editorIsOpen`

Reply:
This object is listd under editor/context, so `&& editorIsOpen` should be removed.